homework/style.css

```diff
@@ -1,3 +1,123 @@
 .alert-error {
```

Comment:
Your app is responsive, but the select element seems to run off the screen:

![image](https://user-images.githubusercontent.com/2788771/59962166-0b9eff80-94e2-11e9-9efa-286eb2cdf244.png)